dss/api/files.py

```diff
@@ -112,6 +114,20 @@ def get_helper(uuid: str, replica: Replica, version: str=None, token: str=None):
 def _verify_checkout(
         replica: Replica, token: typing.Optional[str], file_metadata: dict, blob_path: str,
 ) -> typing.Tuple[str, bool]:
+    try:
+        cloud_handle = Config.get_blobstore_handle(replica)
+        hca_handle = Config.get_hcablobstore_handle(replica)
+        last_modified = cloud_handle.get_last_modified_date(replica.checkout_bucket, "files/{}".format(blob_path))
+        stale_date = datetime.datetime.now(pytz.utc) - datetime.timedelta(days=int(os.environ['DSS_BLOB_PUBLIC_TTL']))
```

Comment:
why not just `datetime.now(timezone.utc)`?

Reply:
Also, I don't think `stale_date` is an appropriate description for what this variable holds.

Reply:
Oh didn't know about that. Agree that we shouldn't be adding dependencies if there are native ways to do it. And you're right - changing to `stale_before_date`

Reply:
Alternatively, you can calculate the time it's going to be stale (last modified time + offset), and test that against the current date.